worlds/stardew_valley/mods/mod_regions.py

```diff
@@ -0,0 +1,96 @@
+from dataclasses import dataclass, field
+from ..data.entrance_data import DeepWoodsEntrance, EugeneEntrance, \
+    JasperEntrance, AlecEntrance, YobaEntrance, JunaEntrance
+from ..data.region_data import SVRegion, DeepWoodsRegion, EugeneRegion, JasperRegion, \
+    AlecRegion, YobaRegion, JunaRegion
+from ..general_classes import RegionData, ConnectionData, RandomizationFlag, ModData
+from .mod_data import ModNames
+from typing import List, Dict
+
+# Making this to reduce clutter from the regions file
+
+# DeepWoods Data
+deep_woods_regions = [
+    RegionData(SVRegion.farm, [DeepWoodsEntrance.use_woods_obelisk]),
+    RegionData(SVRegion.secret_woods, [DeepWoodsEntrance.secret_woods_to_deep_woods]),
+    RegionData(DeepWoodsRegion.main_lichtung, [DeepWoodsEntrance.deep_woods_house,
+                                               DeepWoodsEntrance.deep_woods_depth_10]),
+    RegionData(DeepWoodsRegion.abandoned_home),
+    RegionData(DeepWoodsRegion.floor_10, [DeepWoodsEntrance.deep_woods_depth_30]),
+    RegionData(DeepWoodsRegion.floor_30, [DeepWoodsEntrance.deep_woods_depth_50]),
+    RegionData(DeepWoodsRegion.floor_50, [DeepWoodsEntrance.deep_woods_depth_70]),
+    RegionData(DeepWoodsRegion.floor_70, [DeepWoodsEntrance.deep_woods_depth_90]),
+    RegionData(DeepWoodsRegion.floor_90, [DeepWoodsEntrance.deep_woods_depth_100]),
+    RegionData(DeepWoodsRegion.floor_100)
+]
+
+deep_woods_entrances = [
+    ConnectionData(DeepWoodsEntrance.use_woods_obelisk, DeepWoodsRegion.main_lichtung),
+    ConnectionData(DeepWoodsEntrance.secret_woods_to_deep_woods, DeepWoodsRegion.main_lichtung),
+    ConnectionData(DeepWoodsEntrance.deep_woods_house, DeepWoodsRegion.abandoned_home,
+                   flag=RandomizationFlag.NON_PROGRESSION | RandomizationFlag.LEAD_TO_OPEN_AREA),
```

Comment:
I don't believe this entrance leads to open area. You cannot reach the town, forest, mountain, from it, unless you get an axe.

This flag is specifically used to mean that accessing this entrance allows the player into an open world that can allow progression regardless of their received items